src/styles/images/index.md.njk

```diff
@@ -8,53 +8,72 @@ layout: layout-pane.njk
+Alternative text, or alt text, is read out by screen readers or displayed onscreen — usually when images do not load or have been disabled in browser settings.
 
-{{ example({group: "styles", item: "images", example: "representative", html: false, open: false, size: "l"}) }}
+All images should include the alt attribute, even if it’s left empty without alt text (`alt=""`).
 
+#### When to use alt text
 
-## Alternative text
+Only add alt text when you need to describe visual details you cannot practically include in the written content.
 
-This guidance is for government teams that build online services. [To learn about publishing alternative text in GOV.UK mainstream content, go to GOV.UK's 'Images' guidance.](https://www.gov.uk/guidance/content-design/images)
+There are some cases when an empty `alt=""` attribute is more appropriate. These include:
 
-Alternative text, or alt text, is read out by screen readers or displayed if an image does not load or if images have been switched off.
+- a decorative image that does not include important content
+- an icon with a text label, where the alt text would repeat the text label
+- an image used in a link, where the image is not needed to understand the link
 
-All images, except decorative images, must have alt text that:
+To decide if an image needs alt text, see the [Web Accessibility Initiative alt decision tree](https://www.w3.org/WAI/tutorials/images/decision-tree/).
+
+#### Writing good alt text
+
+Good alt text:
 
 - tells people what information the image provides
 - describes the content and function of the image
 - is specific, meaningful and concise
 
 Use normal punctuation, like commas and full stops, so the text is easy to read and understand.
 
-Do not:
+Generally, aim for no more than 2 sentences of alt text.
+
+Do not write alt text that:
 
-- include the name of the photographer or person who created the image
-- start with ‘Image of’, ‘Graphic of’ or ‘Photo of’
-- repeat information from the page
-- include extra information not in the image
+- includes the name of the photographer or person who created the image
+- starts with ‘Image of’, ‘Graphic of’ or common information that would repeat across every image
+- repeats information from the page
+- includes extra information not in the image
 
 {{ example({group: "styles", item: "images", example: "alt-text", html: true, open: true, size: "l"}) }}
 
-Some images do not need descriptions. Use `alt=""` if the image is:
+### Avoid images that contain text
 
-- decorative and does not include important content
-- an icon that has a text label that the alt text would repeat
-- used in a link and the image is not needed to understand the link
+Text becomes unreadable when the user zooms into images, especially PNGs and JPGs. If you need to use an image that contains text, create it as a vector graphic (SVG) so that it remains readable when the user zooms in. The alt text should be the same as the text displayed in the image.
```

Comment:
I think we lost the main point of this paragraph during the iterations, so I'll try to add back the part that says 'Avoid images that contain text'. Additionally, it might be clearer if we start by talking about what they should do (use an SVG) before listing why PNG and JPEG aren't great for images with text.

```suggestion
Text in images, especially illustrations, is often better placed as HTML text on the page.

If you need to use an image that contains text, create it as a vector graphic (SVG) so that it remains readable when the user zooms in. Text can become pixellated or unreadable when zooming into images, especially PNGs and JPGs.

If contextually important text is displayed in an image, the alt text should include the displayed text.
```

That last sentence likely needs a bit of content design love though.

Reply:
@davidc-gds Would "HTML text" include anything more than written content and things like panels? Be good to keep suggestions to what we have within the Design System if possible.

I'm a bit anxious about the last line, as it's such an absolute worst case scenario, needs to point to written content first and slightly duplicates content on this page. But maybe we can keep it short enough to work.

``suggestion
Use written content or HTML text (such as in the [Panel component](.../components/panel/)) instead.

If you need to use an image that contains text, create it as a vector graphic (SVG) so that it remains readable when the user zooms in. Text can become pixellated or unreadable when zooming into images, especially PNGs and JPGs.

Any important text you display in an image should be included in the written content — or in alt text if this not practical.
```

Reply:
Hey @calvin-lau-sig7 'written content' would be a better descriptor than 'HTML text' yeah - I was mainly trying to distinguish it from just using the word 'text', since that could be construed to include text that appears in an image and would kinda defeat the purpose of the sentence.
 
So yeah, 'written content' is probably all we need, without the part about a panel.

And I think your version of the last line resolves the concern I had and also avoids presenting the worst-case scenario as normal.  I would just change 'practical' to 'possible' though, because practicality is kind of a matter of opinion and it would be easy to justify heavy use of images with text and say it's not "practical" to transcribe the text or run OCR.

```suggestion
Use written content instead of an image that contains text, wherever possible.

If you need to use an image that contains text, create it as a vector graphic (SVG) so that it remains readable when the user zooms in. Text can become pixellated or unreadable when zooming into images, especially PNGs and JPGs.

Any important text you display in an image should be included in the written content — or in alt text if this not possible.
```

Reply:
Nice! Will commit.